app/web/pages/profile/index.tsx

```diff
@@ -0,0 +1,21 @@
+import { appGetLayout } from "components/AppRoute";
+import { ProfilePage as ProfilePageComponent } from "features/profile";
+import { GetStaticProps } from "next";
+import nextI18nextConfig from "next-i18next.config";
+import { serverSideTranslations } from "next-i18next/serverSideTranslations";
+
+export const getStaticProps: GetStaticProps = async ({ locale }) => ({
+  props: {
+    ...(await serverSideTranslations(
+      locale ?? "en",
+      ["global", "profile"],
+      nextI18nextConfig
+    )),
+  },
+});
+
+export default function ProfilePage() {
+  return <ProfilePageComponent tab="about" />;
```

Comment:
```suggestion
  return <ProfilePageComponent />;
```
Wonder if it's better to do this and leave the defaulting to "about" on `ProfilePage` instead, since that's how it's done in the community page. Don't think I have a strong preference either way, as long we do it consistently across all tabbed pages.